test/test-client-server.js

```diff
@@ -1157,7 +1157,61 @@ var tests = [
     },
     what: 'client auto-rejects unrequested, allows requested forwarded-tcpip'
   },
+  { run: function() {
+      var self = this,
+          what = this.what,
+          client,
+          server,
+          wasReply = false,
+          fastrejectSent = false,
+          r;
+
+      function sendAcceptLater(accept) {
+        // Only call accept after reject was called
+        if (fastrejectSent) {
+          accept();
+        } else {
+          process.nextTick(sendAcceptLater, accept);
```

Comment:
This is failing on node v0.10 and node v0.12 because `process.nextTick()` did not accept arguments to pass to the function until iojs and node v4+.